artemis-server/src/main/java/org/apache/activemq/artemis/core/paging/cursor/PagedReferenceImpl.java

```diff
@@ -30,10 +30,9 @@
 import org.apache.activemq.artemis.core.server.ServerConsumer;
 import org.apache.activemq.artemis.core.server.impl.AckReason;
 import org.apache.activemq.artemis.core.transaction.Transaction;
-import org.apache.activemq.artemis.utils.collections.LinkedListImpl;
 import org.jboss.logging.Logger;
 
-public class PagedReferenceImpl extends LinkedListImpl.Node<PagedReferenceImpl> implements PagedReference, Runnable {
```

Comment:
This was an optimization you performed some time ago, to speed up things on the delivering... Queue.references...

Are you sure about this change? Did you validate perf on it?

Reply:
Yes but i was young :)
I didn't know that pages references are never placed on the queue, hence they don't need to extends node

Reply:
<= He had hair back then and care free:P